### DISCUSSION TITLE
What does System Corruption Scan do?

### DISCUSSION THREAD

**Discussion by gitchat1** [ChrisTitusTech/winutil#567] [Q&A]
Can I run it in safe mode? 
DoI have to download anything ahead of time such as a Windows installation media or is everything done by the script?
What happens if the script downloads something and I run out of storage space?

**Comment by techguy16** [comment]
It checks for corrupted files

**Comment by gitchat1** [comment]
Do I need to download anything or does the script do so automatically? What happens if I don't have enough storage space?

**Reply by techguy16** [reply]
No you don't need anything - it uses `sfc` and `dism` (which are built in to Windows)

**Reply by techguy16** [reply]
It doesn't download anything

**Comment by anonyao** [comment]
Corrupt files? + dism or have u chabge it?
